src/server.rs

```diff
@@ -149,12 +164,23 @@ impl Server {
         crdt.write()
             .expect("'crdt' write lock before insert() in pub fn replicate")
             .insert(&entry_point);
-        let window = streamer::default_window();
+
+        let blob_recycler = BlobRecycler::default();
+        let window = match window_entries {
```

Comment:
Copy-paste in one PR!? 👎

Reply:
<sigh>  at this point, yes.

Reply:
will collapse once I grok

Reply:
whacked.  only leader needs a populated window for this PR.